ramandecompy/peakidentify.py

```diff
@@ -600,6 +597,7 @@ def score_sort(row_i, row_j, k, precision):
     sortedscores.sort()
 
     return sortedscores
+
 def process_score(unknown_peaks,known_peaks,k, precision, unknownname, knownname):
     "documentation"
```

Comment:
need doc string here

Reply:
@kernb2 please review comments and update :)

Reply:
made doc string